client/client.go

```diff
@@ -188,6 +188,22 @@ type (
 	// WorkerBuildIDVersionSets is the response for Client.GetWorkerBuildIdCompatibility.
 	WorkerBuildIDVersionSets = internal.WorkerBuildIDVersionSets
 
+	// BuildIDOpAddNewIDInNewDefaultSet is an operation for UpdateWorkerBuildIdCompatibilityOptions
+	// to add a new BuildID in a new default set.
+	BuildIDOpAddNewIDInNewDefaultSet = internal.UpdateBuildIDOpNewSet
```

Comment:
super-nitpicky but this might be easier to read?
```suggestion
	BuildIDOp_AddNewIDInNewDefaultSet = internal.UpdateBuildIDOp_NewSet
```

Reply:
@cretz  preferred it this way. I don't care much either way

Reply:
Please don't use underscores. And ideally share a common prefix.